pages/pipelines/build_exports.md

```diff
@@ -1,12 +1,36 @@
-# Export build data
-
-Each [Buildkite plan](https://buildkite.com/pricing) has a maximum build retention
-period, after which your oldest builds are automatically deleted.
+# Build exports
 
 > 📘 Enterprise feature
-> Build export is only available on an [Enterprise](https://buildkite.com/pricing) plan.
+> Build exports is only available on an [Enterprise](https://buildkite.com/pricing) plan, which has a [build retention](/docs/pipelines/build-retention) period of 12 months.
+
+If you need to retain build data beyond the [retention period](/docs/pipelines/build-retention) in your [Buildkite plan](https://buildkite.com/pricing), you can use the build exports feature.
+
+Build exports enables you to configure your own S3 bucket for Buildkite to export your build data to.
+
+If you do not configure an S3 bucket, the exported build data will be stored for 18 months by Buildkite in case you need it. You cannot access this build data using the Buildkite UI or API, but you can request them from us by contacting support.
+
+## How it works
+Builds older than the build retention limit will be automatically exported as JSON to the S3 bucket you have configured. If you haven't configured a build export bucket, Buildkite will store that build data as JSON in our own S3 bucket for a further 18 months in case you need it.
```

Comment:
A few small style nitpicks + adding a lead-in sentence for the diagram.

```suggestion
Builds older than the build retention limit are automatically exported as JSON to the S3 bucket you have configured. If you haven't configured a bucket for build export, Buildkite stores that build data as JSON in our own S3 bucket for a further 18 months in case you need it. The following diagram outlines this process.
```